src/Infrastructure.VS.UnitTests/ActiveDocumentTrackerTests.cs

```diff
@@ -114,115 +116,85 @@ public void OnSelectionChanged_DoesNotRaiseEvent()
             var result = (testSubject as IVsSelectionEvents).OnSelectionChanged(null, 1, null, null, null, 1, null, null);
             result.Should().Be(VSConstants.S_OK);
 
-            mockEventHandler.VerifyNoOtherCalls();
-        }
-
-        [TestMethod]
-        public void OnElementValueChanged_NewValueIsNull_DoesNotRaiseEvent()
-        {
-            var result = SimulateElementValueChanged(VSConstants.VSSELELEMID.SEID_WindowFrame, null);
-            result.Should().Be(VSConstants.S_OK);
-
-            mockEventHandler.VerifyNoOtherCalls();
+            CheckEventNotRaised();
         }
 
         [TestMethod]
-        public void OnElementValueChanged_NonWindowFrameElement_DoesNotRaiseEvent()
+        [DataRow(VSConstants.VSSELELEMID.SEID_PropertyBrowserSID)]
+        [DataRow(VSConstants.VSSELELEMID.SEID_ResultList)]
+        [DataRow(VSConstants.VSSELELEMID.SEID_StartupProject)]
+        [DataRow(VSConstants.VSSELELEMID.SEID_UndoManager)]
+        [DataRow(VSConstants.VSSELELEMID.SEID_UserContext)]
+        [DataRow(VSConstants.VSSELELEMID.SEID_WindowFrame)]
+        public void OnElementValueChanged_NonDocumentElementId_DoesNotRaiseEvent(VSConstants.VSSELELEMID elementId)
         {
-            var selectedFrame = CreateMockFrame(__WindowFrameTypeFlags.WINDOWFRAMETYPE_Document);
```

Comment:
We no longer need the tests that create frames with different kinds of "frame type".